src/index.spec.js

```diff
@@ -12,15 +18,15 @@ describe('gulp-inject-partials', function(){
 	var logOutput = [];
 
 	beforeEach(function () {
-		log = gutil.log;
+		log = fancyLog.log;
```

Comment:
I'd recommend to use `fancyLog.info` (everywhere in this package) because of this kind of test relying on Node's cache to patch shared dependencies. (`fancyLog.log` does not exist)

I personally consider a bit brittle so I exposed the logger on the main module [in another PR](https://github.com/sindresorhus/gulp-debug/pull/41). Finally, a solution would be to use a `proxy-require` to stub `fancy-log`.